src/pages/overview/pdf-electronic-seal-api/index.md

```diff
@@ -34,3 +34,11 @@ PDF Electronic Seal API currently supports the following TSPs : <!-- REFERENCES
 * [Entrust](https://www.entrust.com/pdf-signing-certificates/)
 * [GlobalSign](https://www.globalsign.com/en/digital-signatures)
 * [TrustPro](https://www.trustpro.eu/)
+
+## Supported Timestamping Authorities
+
+A Timestamping Authority (TSA) is a trusted entity that employs the [RFC3161](https://www.ietf.org/rfc/rfc3161.txt) protocol to generate and attach cryptographic timestamps to digital data, certifying its existence at a specific time, thus enhancing data integrity and legal validity.
+
+The PDF Electronic Seal API supports this trusted timestamping while sealing the document using one of the supported TSA. Many popular TSAs like Intesi Group, DigiCert, GlobalSign, Seiko Timestamp Service, InfoCert Qualified TSA etc. are supported. Adobe Qualified Timestamp Service is also allow-listed to use with Electronic Seal Service. If the use of an additional TSA is required beyond the ones already supported, it can be facilitated during the customer onboarding process. <!-- REFERENCES https://helpx.adobe.com/in/sign/using/custom-time-stamp-providers.html -->
```

Comment:
The PDF Electronic Seal API supports this trusted timestamping -> The PDF Electronic Seal API supports the trusted timestamping

Reply:
Many popular TSAs like Intesi Group, DigiCert, GlobalSign, Seiko Timestamp Service, InfoCert Qualified TSA etc. are supported. -> Supported TSAs are: 

Provide a bullet list.

Reply:
If support for any other TSA is required, reach out to us through customer support.